src/Task/FluentMigrationTask.php

```diff
@@ -205,18 +206,38 @@ function ($className) {
         );
     }
 
+    /**
+     * Return the fields that actually exist on a table
+     *
+     * @param string $table
+     * @return array
+     */
+    protected function getExistingFields($table)
+    {
+        $query = sprintf("SHOW COLUMNS from %s", $table);
+        try {
+            return DB::query($query)->column('Field');
+        } catch(\Exception $ex) {
+            return [];
+        }
+    }
+
     /**
      * @param $fields
      * @param $locale
      * @return string
      */
-    protected function buildSelectFieldList($sourceTable, $fields, $locale)
+    protected function buildSelectFieldList($sourceTable, $fields, $locale, $existingFields)
     {
         return implode(
             ', ',
             array_map(
-                function ($field) use ($locale, $sourceTable) {
-                    return sprintf('COALESCE(`%s_%s`, `%s`.`%s`) AS `%s`', $field, $locale, $sourceTable, $field, $field);
```

Comment:
Pre-existing issue, but I highly prefer ansi-quoted table identifiers (`"`) not backticks when forming SQL queries. One is standard, other is mysql specific.